Jenkinsfile

```diff
@@ -91,10 +110,11 @@ pipeline {
                 }
             }
         }
-        stage('Publish') {
+
+        stage('Publish if Release') {
             when {
                 expression {
-                    return env.shouldBuild != "false" && env.shouldTest != "false"
```

Comment:
should we keep these old checks still?  why do they no longer matter?

Reply:
`shouldBuild` and `shouldTest` only apply to PR builds since they come from comments in the PR so they are NA for the release builds.